db/schema.rb

```diff
@@ -33,5 +33,22 @@
     t.index ["company_id"], name: "index_jobs_on_company_id", using: :btree
   end
 
+  create_table "jobs_tags", force: :cascade do |t|
+    t.integer  "job_id"
```

Comment:
Me too. It wasn't part of the exercise. I'm also checking migrations and there's nothing there that would populate that to the schema.
I know that when I was feeling a little lost I was trying to remember what MANY to MANY relationships were used back when we did Job Tracker and I was thinking about how Tags and jobs had that relationship, and how they might be implemented in this scenario. However... unless I made some silly type while I was thinking about that, I can't think of why it would be in there unless it was in the repo when I cloned it down right before I took the test.
No idea.